test/benchmark/BenchmarkIngestWithNuclio_test.go

```diff
@@ -0,0 +1,148 @@
+package benchmark
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/nuclio/nuclio-sdk-go"
+	"github.com/nuclio/nuclio-test-go"
+	"github.com/pkg/errors"
+	"github.com/v3io/v3io-go-http"
+	"github.com/v3io/v3io-tsdb/config"
+	"github.com/v3io/v3io-tsdb/pkg/tsdb"
+	"github.com/v3io/v3io-tsdb/pkg/utils"
+	"github.com/v3io/v3io-tsdb/test/benchmark/common"
+	"io/ioutil"
+	"log"
+	"os"
+	"testing"
+	"time"
+)
+
+const defaultDbName = "db0"
+
+func BenchmarkIngestWithNuclio(b *testing.B) {
+	log.SetFlags(0)
+	log.SetOutput(ioutil.Discard)
+	testStartTimeNano := time.Now().UnixNano()
+
+	var count = 0 // count real number of samples to compare with query result
+
+	testConfig, err := common.LoadBenchmarkIngestConfigFromData()
+	if err != nil {
+		panic(err)
+	}
+
+	v3ioConfigFile := os.Getenv("V3IO_TSDBCFG_PATH")
+	v3ioConfig, err := config.LoadConfig(v3ioConfigFile)
+	if err != nil {
+		panic(errors.Wrap(err, fmt.Sprintf("Failed to load config from file %s", v3ioConfigFile)))
```

Comment:
> Error strings should not start with a capital letter because they'll often be prefixed before printing:
https://github.com/golang/go/wiki/Errors
(yes, I know this is violated in a bunch of other places in the source code)